### PR TITLE
Add android_test layer

### PR DESCRIPTION
**Pull request by alex1545**

### CODE REVIEW

File: layers/ubuntu1604/android_test/BUILD [hunk 1]
```diff
@@ -42,3 +48,25 @@ download_pkgs(
...
     ],
 )
+
+language_tool_layer(
+    name = "android-test-ltl",
+    base = ":ubuntu1604_i386.tar",
+    installables_tar = "@ubuntu1604_android_test_debs//file",
+)
+
+toolchain_container(
```
Comment (xingao267) [inline layers/ubuntu1604/android_test/BUILD]: Move this target to be above language_tool_layer target.

File: layers/ubuntu1604/android_test/deps_spec.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+# limitations under the License.
+
+revisionsFilePath: "layers/ubuntu1604/android_test/revisions.bzl"
+
+gcsDeps:
+
+  # The debian packages tarball pulled from the GCS bucket required for the Ubuntu1604 android test layer.
+  - name: "DEBS_TARBALL"
+    bucket: "layer-deps"
+    versionRegex: "\\d{10}"
```
Comment (xingao267) [inline layers/ubuntu1604/android_test/deps_spec.yaml]: {10,}

File: layers/ubuntu1604/android_test/BUILD (hunk 1 above)
Comment (alex1545) [reply layers/ubuntu1604/android_test/BUILD]: Done

File: layers/ubuntu1604/android_test/deps_spec.yaml (hunk 2 above)
Comment (alex1545) [reply layers/ubuntu1604/android_test/deps_spec.yaml]: Done

**Pull request merged**